package.json

```diff
@@ -20,6 +20,7 @@
     "next": "14.0.4",
     "react": "^18",
     "react-dom": "^18",
+    "react-icons": "^4.12.0",
```

Comment:
We're already using lucide-react for icons. 
Could you pls check if the icon you're looking for is available there? 

and if not, adding a whole package for a single icon seems unreasonable. maybe a single svg would suffice.

Reply:
just a minute I will check